### ISSUE TITLE
Theme is not affecting Client Side Decoration of HeaderBar?

### ISSUE THREAD

**Issue by luxe** [numixproject/numix-gtk-theme#636]
Once installing the theme, I noticed that it was only affecting _some_ of the Ubuntu windows. It seems that applications have their own window paneling, that I am unable to change through the theme.

For example:

Notice how `gedit` has the default Ubuntu buttons,
while the GNOME file manager is using the applied `numix` theme:  

![theme](https://cloud.githubusercontent.com/assets/1312081/21571342/4fd678f4-ce9c-11e6-8c93-6820e6439bfa.JPG)  

Is there a way I can get `gedit` and other applications to match my existing theme?  
I want all the windows to look the same.   

Reading more into it, I found a relevant post:  
http://stackoverflow.com/questions/28650646/what-is-client-side-decoration
However, I have been unable to fix the problem.  
Looking at other screenshots from issues in this project, I can see that this isn't a problem for other users.   
Any thoughts on what I can do?  

**Additional info:**  
I am on `Ubuntu 16.04 LTS`, and have gone to the `gnome-flashback`. I'm also using the `gnome-tweak-tool` to apply the GTK+ theme.  
It seems that I'm on GTK 3.0 (at least that's what I'm changing in the `numix` theme to see relevant changes).

**Comment by palob** [comment]
That's strange. Actually ``gedit`` is one of the apps the Ubuntu folks patched so as not to use client-side decorations. The gedit window is actually server-side decorated.
Nautilus (GNOME Files) is another one they patched so I wonder why it does use CSDs (a headerbar) in your screenshot.

Anyway, have you got some other themes installed in order to check whether it is a Numix-only issue?

**Comment by luxe** [comment]
I've tried some other themes now (Adwaita, HighContrast, Orion, Radiance).  
They all have the same problem. I suppose its something to do with my ubuntu configuration, although this is a fresh install with minimal changes made.  

Are most applications suppose to inherit changes from the theme?  
Most of my applications do not:  
gnome-terminal, firefox, calculator, dconf-editor,  to name a few.  

Programs that are changing seem to  be:  
file-manager, gnome-tweak-tool, syslog

**Comment by palob** [comment]
I'm no expert in Ubuntu questions so I'm at a loss as to why that's happening. However from what you tell the theme is only applied to CSD apps.

Can you fire up ``dconf-editor`` and navigate to ``/org/gnome/desktop/wm/preferences/`` and look up what's the value for ``theme``?
Same for ``/org/gnome/desktop/interface`` and the value for ``gtk-theme``?

**Comment by luxe** [comment]
Under  `org.gnome.desktop.wm.preferences`, the attribute states   
`DEPRECATED: This key is deprecated and ignored`.  

It was set to `Adwaita`, but changing it to `Numix` had no effect, so I assume it is indeed being ignored.
The property at `org.gnome.desktop.interface gtk-theme`, and it already set to Numix.  
The `gnome-tweak-tool` is changing that property.

I finally found the problem:  
`org.gnome.metacity.theme` was not being set.  
I'm not sure how to change this property without doing it through the editor manually, but changing it to `Numix` has adjusted all my remaining windows.   

These are the two places it seems I needed to apply the theme for it to affect all the windows:  
```
gsettings set org.gnome.desktop.interface gtk-theme 'Numix';
gsettings set org.gnome.metacity 'Numix';
```

Thank you for your help.
